server/routerlicious/packages/services/src/redis.ts

```diff
@@ -56,6 +56,24 @@ export class RedisCache implements ICache {
         }
     }
 
+    public async incr(key: string): Promise<number> {
+        try {
+            return this.client.incr(key);
```

Comment:
Maybe directly using `incrBy` api and default set to 1?

Reply:
You mean directly use incrBy in Lambda? We need to abstract redis dependency, and hence this is required. Also, later on we might have need this in other places too, for adding more telemetry metrics.

Reply:
It is fine to leave this resolved. However, there are only two major distributed cache, redis and MemecacheD. They both support `incrBy` and I believe using `incrBy` is more flexible than `incrOne`

As for abstruset dependency away, I agree with you. But still `IncrBy` is a more flexible abstract interface than `IncrOne`, customer can just call `IncrBy(1)` to achieve the `IncrOne` behavior